src/js/components/DataTable/stories/data.js

```diff
@@ -70,6 +70,27 @@ for (let i = 0; i < 40; i += 1) {
 }
 
 export const DATA = [
+  {
+    name: 'zoey',
+    location: '',
```

Comment:
Is there a reason we're not filling out the other properties with more interesting values? I ask because the other stories sharing this data are looking a little off.

Reply:
Hi @ericsoderberghp, I think my point in adding these items to the data was to demonstrate case-insensitive sorting (or lack thereof). Prior to this, only initial-capped values were used, so this was a blind-spot. If the data is "perfect", then little bugs can creep in that don't become obvious until they encounter real-world edge cases. If'd you'd like to see more interesting values so that other stories look better, let's please make sure we don't eliminate some of the intentional flaws I introduced by this change.

Reply:
That's fine. In that case, maybe we should create a custom `data` value for this story.

Reply:
Hey @ericsoderberghp, okay. I've just pushed the change. Please review.